mobile/src/stateMachine/views/healthData/healthConsent.tsx

```diff
@@ -0,0 +1,31 @@
+import { observer } from 'mobx-react';
+import {Alert } from 'react-native';
+import { ViewState } from '../base';
+import AppController from 'src/controllers';
+import { ScenarioTriggers } from '../../abstractions';
+import logger from 'common/logger';
+
+
+@observer
+export class HealthConsentView extends ViewState {
+    async start() {
+        this.showModal({
+            title: 'We need your health data to build a better personalized experience for you.',
+            message: 'Would you like to grant permission?',
```

Comment:
Either add a second button here allowing the user to decline or change the message.